src/scss/_mixins.scss

```diff
@@ -243,16 +239,20 @@
 		background-color: $background;
 	}
 
-	@if not $is-light {
+	@if not $link-contrasts-with-background {
```

Comment:
Could we style the links when there is enough contrast with `oTypographyLink`? As mentioned in the other PR comment, Origami components are usually self contained. Alternatively we could style anchors in our demo CSS and document in the readme that to use `o-topper` `a` tags must be styled using o-typography.

Reply:
Could this be done as a follow up pull request? I wanted to avoid this as it's got potential for conflicts whereas your team will have more of an idea what to avoid.